scripts/appscript.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# Required vars
+SALT_ARCHIVE=${salt_content_archive}
+SALT_DIR="/srv/salt"
+ARCHIVE_FILE_NAME="salt_formula.zip"
+
+# Standard aws envs
+export AWS_DEFAULT_REGION=$(curl -sSL http://169.254.169.254/latest/meta-data/placement/availability-zone | sed 's/.$//')
```

Comment:
recommend splitting into two lines, to avoid masking errors...
```suggestion
AWS_DEFAULT_REGION="$(curl -sSL http://169.254.169.254/latest/meta-data/placement/availability-zone | sed 's/.$//')"
export AWS_DEFAULT_REGION
```